charts/opensearch-operator/templates/opensearchclusters.opensearch.opster.io-crd.yaml

```diff
@@ -53,6 +53,10 @@ spec:
                 type: object
               dashboards:
                 properties:
+                  additionalConfig:
+                    additionalProperties:
+                      type: string
+                    type: object
```

Comment:
This PR looks good to me.

Repo owner,
Other than this missing spec, I also noticed that the content in opensearch.opster.io_opensearchclusters.yaml is different from charts/opensearch-operator/templates/opensearchclusters.opensearch.opster.io-crd.yaml.

Are those two files suppose to have the same content?

```
$ diff charts/opensearch-operator/templates/opensearchclusters.opensearch.opster.io-crd.yaml opensearch-operator/config/crd/bases/opensearch.opster.io_opensearchclusters.yaml 
55a56,60
>                   additionalConfig:
>                     additionalProperties:
>                       type: string
>                     description: Additional properties for opensearch_dashboards.yaml
>                     type: object
58,67d62
<                   opensearchCredentialsSecret:
<                     description: Secret that contains fields username and password
<                       for dashboards to use to login to opensearch, must only be supplied
<                       if a custom securityconfig is provided
<                     properties:
<                       name:
<                         description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
<                           TODO: Add other useful fields. apiVersion, kind, uid?'
<                         type: string
<                     type: object
84a80,89
>                   opensearchCredentialsSecret:
>                     description: Secret that contains fields username and password
>                       for dashboards to use to login to opensearch, must only be supplied
>                       if a custom securityconfig is provided
>                     properties:
>                       name:
>                         description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
>                           TODO: Add other useful fields. apiVersion, kind, uid?'
>                         type: string
>                     type: object
1095a1101,1213
>                     env:
>                       items:
>                         description: EnvVar represents an environment variable present
>                           in a Container.
>                         properties:
>                           name:
>                             description: Name of the environment variable. Must be
>                               a C_IDENTIFIER.
>                             type: string
>                           value:
>                             description: 'Variable references $(VAR_NAME) are expanded
>                               using the previously defined environment variables in
>                               the container and any service environment variables.
>                               If a variable cannot be resolved, the reference in the
>                               input string will be unchanged. Double $$ are reduced
>                               to a single $, which allows for escaping the $(VAR_NAME)
>                               syntax: i.e. "$$(VAR_NAME)" will produce the string
>                               literal "$(VAR_NAME)". Escaped references will never
>                               be expanded, regardless of whether the variable exists
>                               or not. Defaults to "".'
>                             type: string
>                           valueFrom:
>                             description: Source for the environment variable's value.
>                               Cannot be used if value is not empty.
>                             properties:
>                               configMapKeyRef:
>                                 description: Selects a key of a ConfigMap.
>                                 properties:
>                                   key:
>                                     description: The key to select.
>                                     type: string
>                                   name:
>                                     description: 'Name of the referent. More info:
>                                       https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
>                                       TODO: Add other useful fields. apiVersion, kind,
>                                       uid?'
>                                     type: string
>                                   optional:
>                                     description: Specify whether the ConfigMap or
>                                       its key must be defined
>                                     type: boolean
>                                 required:
>                                 - key
>                                 type: object
>                               fieldRef:
>                                 description: 'Selects a field of the pod: supports
>                                   metadata.name, metadata.namespace, `metadata.labels[''<KEY>'']`,
>                                   `metadata.annotations[''<KEY>'']`, spec.nodeName,
>                                   spec.serviceAccountName, status.hostIP, status.podIP,
>                                   status.podIPs.'
>                                 properties:
>                                   apiVersion:
>                                     description: Version of the schema the FieldPath
>                                       is written in terms of, defaults to "v1".
>                                     type: string
>                                   fieldPath:
>                                     description: Path of the field to select in the
>                                       specified API version.
>                                     type: string
>                                 required:
>                                 - fieldPath
>                                 type: object
>                               resourceFieldRef:
>                                 description: 'Selects a resource of the container:
>                                   only resources limits and requests (limits.cpu,
>                                   limits.memory, limits.ephemeral-storage, requests.cpu,
>                                   requests.memory and requests.ephemeral-storage)
>                                   are currently supported.'
>                                 properties:
>                                   containerName:
>                                     description: 'Container name: required for volumes,
>                                       optional for env vars'
>                                     type: string
>                                   divisor:
>                                     anyOf:
>                                     - type: integer
>                                     - type: string
>                                     description: Specifies the output format of the
>                                       exposed resources, defaults to "1"
>                                     pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
>                                     x-kubernetes-int-or-string: true
>                                   resource:
>                                     description: 'Required: resource to select'
>                                     type: string
>                                 required:
>                                 - resource
>                                 type: object
>                               secretKeyRef:
>                                 description: Selects a key of a secret in the pod's
>                                   namespace
>                                 properties:
>                                   key:
>                                     description: The key of the secret to select from.  Must
>                                       be a valid secret key.
>                                     type: string
>                                   name:
>                                     description: 'Name of the referent. More info:
>                                       https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
>                                       TODO: Add other useful fields. apiVersion, kind,
>                                       uid?'
>                                     type: string
>                                   optional:
>                                     description: Specify whether the Secret or its
>                                       key must be defined
>                                     type: boolean
>                                 required:
>                                 - key
>                                 type: object
>                             type: object
>                         required:
>                         - name
>                         type: object
>                       type: array
1097a1216,1219
>                     labels:
>                       additionalProperties:
>                         type: string
>                       type: object
```